pyproject.toml

```
@@ -0,0 +1,48 @@
+[build-system]
+requires = ["setuptools >= 77", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pals_schema"
+version = "25.08"
+dependencies = [
+  "pydantic",
+  "pyyaml",
+  "toml",
+]
+requires-python = ">=3.10"
+authors = [
+  { name="Axel Huebl", email="axelhuebl@lbl.gov" },
+  { name="Edoardo Zoni", email="ezoni@lbl.gov" },
+]
+maintainers = [
+  { name="Axel Huebl", email="axelhuebl@lbl.gov" },
+  { name="Edoardo Zoni", email="ezoni@lbl.gov" },
+]
+description = "Python implementation for the Particle Accelerator Lattice Standard (PALS)"
+readme = "README.md"
+license = "BSD-3-Clause-LBNL"
+license-files = ["LICEN[CS]E*"]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Framework :: Pydantic :: 2",
+  "Intended Audience :: Developers",
+  "Intended Audience :: Science/Research",
+  "Natural Language :: English",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.10",
```

This one here is soft meta-data, so one can update it manually or just say 3 and never worry again :D
```suggestion
   "Programming Language :: Python :: 3.9",
    "Programming Language :: Python :: 3.10",
```